components/alert/index.tsx

```diff
@@ -154,11 +173,21 @@ const Alert: AlertInterface = ({
       [`${prefixCls}-no-icon`]: !isShowIcon,
       [`${prefixCls}-banner`]: !!banner,
       [`${prefixCls}-closable`]: isClosable,
+      [`${prefixCls}-actionable`]: actionable,
+      [flexLayoutCls]: !actionable,
       [`${prefixCls}-rtl`]: direction === 'rtl',
     },
     className,
   );
 
+  const OffsetLayout = ({ offset, children }: OffsetLayoutProps) => {
+    if (offset)
+      return (
+        <div className={classNames(`${prefixCls}-offset-wrap`, flexLayoutCls)}>{children}</div>
+      );
+    return <>{children}</>;
```

Comment:
No need for fragment.

Reply:
wrap `children` with `fragment` is valid for simply return `children`. 
Just a trick, I have no good way to deal with this recently.
Can you give me a hint?

Reply:
children as `React.ReactNode`?

Reply:
There's no need for wrapping a fragment outside, since it's passed by user. In typescript it should be ReactNode and in js there will be no guarantee for what will be passed in.

Reply:
`'OffsetLayout' cannot be used as a JSX component`
if return separate children